cmd/swarm/main.go

```diff
@@ -218,12 +218,21 @@ The output of this command is supposed to be machine-readable.
 `,
 		},
 		{
-			Action:    upload,
+			Action:    nonEncryptedUpload,
 			Name:      "up",
 			Usage:     "upload a file or directory to swarm using the HTTP API",
 			ArgsUsage: " <file>",
 			Description: `
 "upload a file or directory to swarm using the HTTP API and prints the root hash",
+`,
+		},
+		{
+			Action:    encryptedUpload,
```

Comment:
why a new command needed, why not a flag as we agreed?

Reply:
@zelig Fixed in a0ae2a37dd8f064c6e4d60cc5e8a84ea027f234f